packages/web/src/CountdownCircleTimer.tsx

```diff
@@ -17,7 +17,12 @@ const CountdownCircleTimer = (props: Props) => {
 
   return (
     <div style={getWrapperStyle(size)}>
-      <svg width={size} height={size} xmlns="http://www.w3.org/2000/svg">
+      <svg
+        viewBox={`0 0 {size} {size}`}
```

Comment:
Hey @blmage, good catch! That makes sense, aren't you missing the `$` before the size here?

Reply:
I guess we will need the same logic for the mobile package as well :)

Reply:
Oops, you're right! Should be all good now for both packages.